__tests__/__snapshots__/plugin.test.js.snap

```diff
@@ -121,9 +121,9 @@ exports[`babel plugin supports SequenceExpression, with objectHook 1`] = `
 function App() {
   (0, useAnimatedGestureHandler)({
     onStart: function () {
-      var _e = [new Error(), 1, -20];
+      const _e = [new Error(), 1, -20];
```

Comment:
Previously we had `var` but now it's `const`, is this expected and are we sure it doesn't break anything?